Lesson_3.java

```diff
@@ -0,0 +1,25 @@
+public class Lesson_3 {
+
+    public static void longWord() {
+        String s1 = "Mother";
```

Comment:
Please do not use specific values in a method body. String word1, String word2, String word3 - place to method parameters.
Specific values ("Mother", "Farther", ...) will be arguments when you use your method in the main method

Reply:
public static void maxLength(String s1, String s2, String s3) {
        int a = s1.length();
        int b = s2.length();
        int c = s3.length();
        if (a > b && a > c) {
            System.out.println("result:\n" + s1);
        } else if (b > a && b > c) {
            System.out.println("result:\n" + s2);
        } else if (c > a && c > b) {
            System.out.println("result:\n" + s3);
        } else {
            System.out.println("result:\n" + "There are no the max-length word");
        }